file-server/src/test/java/se/fortnox/reactivewizard/fileserver/FileServerTest.java

```diff
@@ -0,0 +1,35 @@
+package se.fortnox.reactivewizard.fileserver;
+
+import io.reactivex.netty.protocol.http.server.MockHttpServerRequest;
+import org.junit.Test;
+import rx.Observable;
+import se.fortnox.reactivewizard.mocks.MockHttpServerResponse;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+public class FileServerTest {
+
+    @Test
+    public void readsFile() {
+        FileServerConfig config = new FileServerConfig();
+        config.setPath(".");
+        FileServerRequestHandler fileServer = new FileServerRequestHandler(config);
+        MockHttpServerResponse response = new MockHttpServerResponse();
+        Observable<Void> result = fileServer.handle(new MockHttpServerRequest("/pom.xml"), response);
```

Comment:
Perhaps add a test that where the file is larger than the buffer of FileUtil.read? Or have the buffer configurable